Runmon/24_White2.yaml

```diff
@@ -192,7 +192,7 @@ moves:
     - [Strength]
     - [Bonemerang]
     - [Brave Bird]
-    - [Hidden Power]
+    - [Fiery Dance]
```

Comment:
```suggestion
    - [Lava Plume]
```
Fiery Dance is gen 5

Reply:
oh shit i typed the wrong thing Jebaited good catch